apps-rendering/src/components/NewsletterSignup/PrivacyWording.tsx

```diff
@@ -0,0 +1,76 @@
+// ----- Imports ----- //
+
+import { css } from '@emotion/react';
+import { neutral, text, textSans } from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import type { FC } from 'react';
+
+// ----- Component ----- //
+
+interface Props {
+	notUsingCaptcha?: boolean;
+}
+
+const termsStyle = css`
+	${textSans.xxsmall({ lineHeight: 'tight' })}
+	color: ${text.supporting};
```

Comment:
I think `text.supporting` was deprecated in v4 of Source? The closest equivalent I can find is `neutral[46]`.

Reply:
it doesn't seem to be marked as deprecated (but there is a warning in the docs about the colour contrast on some backgrounds)- maybe it was restored at some point?

Reply:
> it doesn't seem to be marked as deprecated

It won't appear as deprecated yet in VSCode because I think the deprecation warning was added to a slightly later version of `source-foundations` than the one we're on in DCR/AR. However if you go to the docs for `source-foundations` it's in the list of deprecated colour tokens for v4.

Reply:
👍  thanks, I'll change to neutral[46] - https://guardian.github.io/source/?path=/docs/packages-source-foundations-palette--palette